packages/react/src/components/Listbox/ListboxOption.tsx

```diff
@@ -68,14 +77,11 @@ const ListboxOption = forwardRef<HTMLElement, ListboxOptionProps>(
           return [...options, option];
         }
 
-        for (const opt of options) {
+        for (let i = 0; i < options.length; i++) {
```

Comment:
Why the change here?

Reply:
It's just easier, previously we had line `const index = options.indexOf(opt);` inside of the loop, so why not have this index already when iterating?

Reply:
I'm partially nitpicking here because it's not clear why the change was made and isn't conventionally how iterations have been performed elsewhere. I understand this can be subjective, but given there's not a problem that's being identified it would be better to leave this untouched unless we're fixing a specific issue.